starter_code/app.js

```diff
@@ -1,21 +1,38 @@
 
 const express = require('express');
```

Comment:
Try to put all the requires at the top and separate them from the res. You will get cleaner code.